app/device/radio.py

```diff
@@ -1,3 +1,34 @@
+import serial
+from time import sleep
+import logging
+import json
+
 class Radio(object):
-    def __init__(self):
-        print "lol"
+    def __init__(self,baud=9600,port='/dev/ttyUSB0'):
+        try:
+            self.ser = serial.Serial(port,baud,timeout=None)
+            logging.info("Radio Initialized")
+        except Exception as e:
+            logging.error('error:{}'.format(e))
+
+    def write(self, raw_data):
+        logging.info(json.dumps(raw_data))
+        try:
+            return self.ser.write(json.dumps(raw_data)+'\n')
+        except Exception as e:
+            logging.error('error: {}, data: {}'.format(e, raw_data))
+
+    def read(self):
+        try:
+            return self.ser.readline()

```

Comment:
don't use `readline`,  it waits for an EOL character, and we want to be able to read 'none'